app/main.py

```diff
@@ -1,3 +1,47 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: (int, float)) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: (Distance, float)) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(km=self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(km=self.km + other)
+
+    def __iadd__(self, other: (Distance, float)) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        elif isinstance(other, (float, int)):
+            self.km += other
+            return self
+
+    def __mul__(self, other: int) -> Distance:
+        return Distance(km=self.km * other)
+
+    def __truediv__(self, other: int) -> Distance:
+        return Distance(km=round((self.km / other), 2))
+
+    def __lt__(self, other: int) -> bool:
```

Comment:
`other` also can be `float` or `Distance`

please fix it here and other places